doc/2/framework/classes/name-generator/generate-random-long-name/index.md

```diff
@@ -0,0 +1,24 @@
+---
+code: true
+type: page
+title: generateRandomLongName
+description: NameGenerator class generateRandomLongName() method
+---
+
+# generateRandomLongName
```

Comment:
Naming proposal: `generateRandomName`

Reply:
I thought it would be better since I added `getRandomName` but this is also cool